### DISCUSSION TITLE
Combining multiple geo-datasets into one

### DISCUSSION THREAD

**Discussion by bitsandbricks** [azavea/raster-vision#1782] [Q&A] [answered]
Hi there!

I have a set of satellite images of the exact same area, taken at different dates (let's say in 2020, 2021, 2022... etc). For each image, I have a geojson labels file, identifying objects of interest present at that date.

Using `SemanticSegmentationRandomWindowGeoDataset` I can easily build a separate training dataset for each date. 

For training purposes, all datasets should be combined into a single pool. 

Is there a way to combine Raster Vision geo-datasets?

**Comment by AdeelH** [comment, marked as answer]
You can combine them together using PyTorch's [`ConcatDataset`](https://pytorch.org/docs/stable/data.html#torch.utils.data.ConcatDataset).
